specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2023-10-01-preview/purviewdatamap.json

```diff
@@ -4002,6 +4363,119 @@
           }
         }
       }
+    },
+    "/intelligence/chat": {
```

Comment:
Seems it should be `/intelligence:chat` if an action, not a resource. Same for `skill` below based on its description. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action for the POST action pattern.

Reply:
it is a noun than an action, "chat" here is equal to "conversation" meaning (we checked some competitors to see whether they choose chat or conversation, and finally choose chat)